README.md

```diff
@@ -88,6 +88,25 @@ you're thinking of using `ariadne` to process your compiler's output, why not tr
 - A plethora of other options (tab width, label attach points, underlines, etc.)
 - Built-in ordering/overlap heuristics that come up with the best way to avoid overlapping & label crossover
 
+## Cargo Features
+
+- `"concolor"` enables integration with the [`concolor`](https://crates.io/crates/concolor) crate for global color output
+  control across your application
+- `"auto-color"` enables `concolor`'s `"auto"` feature for automatic color control, useful if `ariadne` is your only
+  dependency using `concolor`
+
+This:
+```TOML
+[dependencies]
+ariadne = { version = "...", features = ["auto-color"] }
+```
+is equivalent to this:
+```TOML
+[dependencies]
+ariadne = { version = "...", features = ["concolor"] }
+concolor = { version = "...", features = ["auto"] }
+```
```

Comment:
I'm not sure whether this example is necessarily justified, unless there's something special about `concolor` that I'm missing?

Reply:
Sort of, yeah. [Library crates aren't supposed to define `concolor`'s features](https://docs.rs/concolor/latest/concolor/#libs) as they're supposed to be up to the binary crate to define for all their dependencies. I added the `auto-color` feature to `ariadne` as I felt it was likely (as in my case) that `ariadne` might be the only dependency that generates colored output and without enabling at least some of `concolor`'s features there's really no point including it at all. The example was just to explain what was going on. I can remove the example and/or the `auto-color` feature if you'd prefer.

Reply:
That makes sense. I wonder whether it's worth clarifying this state of affairs in the README? I think it confused me when first reading it.

Reply:
Updated the README, rebased and fixed some merge conflicts.